### DISCUSSION TITLE
Cannot manage to make it work...

### DISCUSSION THREAD

**Discussion by KevTale** [jeffreylanters/react-unity-webgl#238] [Help Wanted!] [answered]
Hi all,

Sorry for the very non-precise title, I couldnt figure a better one.

Anyway, I've followed the very basic step with Unity 2020.3.18f1 and the react-unity-webgl lib:

1. Unity side, I've created a new project from scratch with URP and immediatally built the project in WebGL with compression disabled
2.  Create a new react app with `npx create-react-app my-app`
3.  `npm install react-unity-webgl@8.x`
4. Put the Build in `public` folder
5. Added the context like so :
```
import React from "react";
import Unity, { UnityContext } from "react-unity-webgl";

const unityContext = new UnityContext({
  loaderUrl: "./build/CustomisationTool.loader.js",
  dataUrl: "./build/CustomisationTool.data",
  frameworkUrl: "./build/CustomisationTool.framework.js",
  codeUrl: "./build/CustomisationTool.wasm",
});

function App() {
  return <Unity unityContext={unityContext} />;
}
export default App;

```
6. `npm run strart` and it just shows a black page and immediatly crash my computer...

As the PC crashed, I couldnt take a screenshot but managed to take a picture, sorry for the horrible quality 😅 

![IMG_20210915_105810](https://user-images.githubusercontent.com/29467803/133406962-c3020e54-52cc-4404-a63b-ec6e9a7ddd27.jpg)

Here's the repo: https://github.com/KevTale/react-unity-webgl
And a netlify domain to check that out: https://hungry-wozniak-9b5b07.netlify.app/

**Note**: the unity project works fine when doing `npx serve` in the Build folder generated by Unity

Thanks for the help!

**Comment by jterdina** [comment, marked as answer]
return (
    <div>
    <Unity
      unityContext={unityContext}
      style={{ width: "640px", height: "480px" }}
    />
    </div>
  );
}

^^^ I had the same problem you are describing. I added the style line to address it.

**Reply by KevTale** [reply]
It works! Thanks :)

**Comment by jeffreylanters** [comment]
Hi. Sorry for my late response. Does your Unity WebGL build work outside of React? You can try this by going into Unity, switching the WebGL build platform en selecting `build and run`. This starts a small web server serving your Unity build. Thanks! Keep me posted.
